samples/video/mt9m114/README.rst

```diff
@@ -0,0 +1,38 @@
+.. _mt9m114:
+
+MT9M114: CMOS image sensor
+########################################
+
+Description
+***********
+
+This sample application simply retrieves frames from MT9M114 sensor via video
```

Comment:
How about:
```
This sample application uses the Video API to retrieve video
frames from the MT9M114 sensor, writes a frame count
message to the console, and then discards the video frame data.
```

Reply:
thanks for your help and review, change applied.